src/components/home-hero/index.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import {Image, StyleSheet, Text, TouchableOpacity, View} from 'react-native';
+import {Movie} from '../../types/movies';
+import {getGenreText, getImageUrl} from '../../utils/moviesUtils';
+import Colors from '../../types/colors';
+import RoundSeparatorIcon from '../../assets/icons/RoundSeparatorIcon';
+import PlayIcon from '../../assets/icons/PlayIcon';
+import InfoIcon from '../../assets/icons/InfoIcon';
+import AddToListIcon from '../../assets/icons/AddListIcon';
+
+const HomeScreenHero = ({movie}: {movie: Movie | null}) => {
+  if (!movie) {
+    return null;
+  }
+  return (
+    <View style={styles.container}>
+      <Image
+        source={{uri: getImageUrl(movie.poster_path)}}
+        style={styles.heroImg}
+      />
+      <Image
+        source={require('../../assets/images/blur-shadow.png')}
+        style={styles.heroBlur}
+        blurRadius={1}
+      />
+      <View style={styles.genreContainer}>
+        {movie.genre_ids.slice(0, 3).map((genre, index) => {
+          return (
+            <>
+              {index > 0 && index <= 2 && (
+                <View key={genre} style={styles.genreSeparator}>
+                  <RoundSeparatorIcon width={10} height={10} />
+                </View>
+              )}
+              <Text key={index} style={[styles.whiteText, styles.genreText]}>
+                {getGenreText(genre)}
+              </Text>
+            </>
+          );
+        })}
+      </View>
+      <View style={styles.movyOriginalContainer}>
+        <Text style={styles.movyOriginalText}>MOVY ORIGINAL</Text>
+      </View>
+      <View style={styles.iconContainer}>
+        <TouchableOpacity style={styles.iconWithText} onPress={() => {}}>
+          <AddToListIcon fill={Colors.white} />
+          <Text style={styles.whiteText}>My List</Text>
+        </TouchableOpacity>
+        <TouchableOpacity style={styles.iconWithText} onPress={() => {}}>
+          <PlayIcon fill={Colors.white} height={43} />
+          <Text style={styles.whiteText}>Play</Text>
+        </TouchableOpacity>
+        <TouchableOpacity style={styles.iconWithText} onPress={() => {}}>
+          <InfoIcon fill={Colors.white} />
+          <Text style={styles.whiteText}>Info</Text>
+        </TouchableOpacity>
```

Comment:
Delegate these buttons to a reusable one as they have exactly the same behaviour